pkg/module/drift/terraform.go

```diff
@@ -93,45 +173,52 @@ func driftTerraform(ctx context.Context, logger hclog.Logger, conn *pgxpool.Conn
 	resData := resources[resName]
 	deepMode := runParams.ForceDeep || (resData.Deep != nil && *resData.Deep)
```

Comment:
is there a point for Deep to be nil ? shouldn't it just be False/True?

Reply:
@roneli It could be unset so that would make it nil. In theory you could only enable this for some of the resources and not all. If it's unset then it would fall back to the wild resource entry. (which can again be `nil`, then which would default to false until you run with the cli --deep flag...)

CLI always overrides the config, but since we need the set/unset logic under a non-wild resource...